### PR TITLE
176: Introducing VSecM Blog page

### PR DESCRIPTION
**Pull request by abhishek44sharma**

### CODE REVIEW

File: docs/_pages/0005-blog.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+title: VSecM Blogs
```
Comment (v0lkan) [inline docs/_pages/0005-blog.md:12]: nitpick: Let’s use “Blog” (singular)

Blog is a place that has multiple blog posts.

“Blogs” implies that we have different kinds of blogs (a design blog, a tech blog, a business blog etc) — which is not the case — Well, at least, not… yet :D .

**Review by v0lkan — CHANGES_REQUESTED** [review]
One minor change request to rename "blogs" to "blog".

Other than that, it’s progress.

I’ll deploy it to prod after merge and see how it looks.

Then maybe we can have follow-up issues.

--

Thanks for putting this together.

File: docs/_pages/0005-blog.md (hunk 1 above)
Comment (abhishek44sharma) [reply docs/_pages/0005-blog.md]: Make sense. 👍

**Comment by abhishek44sharma** [comment]
> 

Done. 👍

**Comment by abhishek44sharma** [comment]
> One minor change request to rename "blogs" to "blog".
> 
> Other than that, it’s progress.
> 
> I’ll deploy it to prod after merge and see how it looks.
> 
> Then maybe we can have follow-up issues.
> 
> --
> 
> Thanks for putting this together.

Done. 👍

**Review by v0lkan — APPROVED** [review]

**Pull request merged**